packages/store/src/storage.ts

```diff
@@ -123,8 +123,8 @@ class Storage {
     debug('change existing package for package %o revision %o', name, revision);
     debug(`change manifest tags for %o revision %o`, name, revision);
     if (
-      !validatioUtils.isObject(metadata.versions) ||
-      !validatioUtils.isObject(metadata[DIST_TAGS])
+      !validationUtils.isObject(metadata.versions) ||
```

Comment:
Note: worth reviewing the implementation of `validationUtils.isObject` probably incomplete or needs cleanup.